### PR TITLE
Load tester that can exercise remote or local testnet 

### PR DESCRIPTION
**Pull request by nearmax**
See loadtester crate for command line example.

Also:
* Fixes chainspec generator, because after adding `alice.near` into accounts only 3 out of 4 nodes were validating;
* Fixes `rpc_user.rs` to timeout on hung connections.

### CODE REVIEW

File: test-utils/testlib/src/user/rpc_user.rs [hunk 1]
```diff
@@ -17,11 +22,18 @@ impl RpcUser {
     fn url(&self) -> String {
         format!("http://{}", self.addr)
     }
+
+    fn client(&self) -> Result<reqwest::Client, String> {
+        reqwest::Client::builder()
+            .timeout(Some(Duration::from_secs(1)))
```
Comment (bowenwang1996) [inline test-utils/testlib/src/user/rpc_user.rs]: Is 1 second enough?

File: test-utils/testlib/src/transactions_executor.rs [hunk 2]
```diff
@@ -135,6 +154,17 @@ impl Executor {
         });
     }
 
+    fn get_nonces(nodes: &Vec<Arc<RwLock<dyn Node>>>) -> RwLock<HashMap<AccountId, u64>> {
+        let mut res = HashMap::new();
+        for node in nodes {
+            let guard = node.read().unwrap();
+            let account_id = guard.account_id().unwrap().clone();
+            let nonce = guard.user().view_account(&account_id).unwrap().nonce;
+            res.insert(account_id, nonce + 100);
```
Comment (bowenwang1996) [inline test-utils/testlib/src/transactions_executor.rs]: Why +100 here?

File: test-utils/testlib/src/user/rpc_user.rs (hunk 1 above)
Comment (nearmax) [reply test-utils/testlib/src/user/rpc_user.rs]: Yes, most of the ping is under 500ms.

File: test-utils/testlib/src/transactions_executor.rs (hunk 2 above)
Comment (nearmax) [reply test-utils/testlib/src/transactions_executor.rs]: Added explanation.

**Pull request merged**